packages/workspace/src/api/WorkspaceConfig.ts

```diff
@@ -1,18 +1,7 @@
-import { ServiceDefinition } from '@scalecube/scalecube-microservice/lib/api';
-
-interface Service {
-  serviceName: string;
-  path: string;
-  definition: ServiceDefinition;
-  config: { [key: string]: any };
-}
-
-interface Component {
-  componentName: string;
-  path: string;
-  config: { [key: string]: any };
-}
+import Service from './Service';
+import Component from './Component';
 
+// @ts-ignore
```

Comment:
Why ts-ignore ?

Reply:
because ts throw an error that it's not used